CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 
 ### Fixed
 * JAX-RS providers with binding annotations will no longer be registered in the resteasy provider factory for the server.
+* Guice-based Beadledom clients will now use the correct time unit for time outs.
```

Comment:
nit: this should be "timeouts".

Reply:
why? time out is not a single word :S

Reply:
I think it is a single word in programming at least.

Reply:
i changed it to time-out: https://github.com/cerner/beadledom/pull/147/commits/f6fb6fc346add9772d79249dfe8bad813ce7f2b6

Reference: https://en.wikipedia.org/wiki/Timeout_(computing)